src/redux/ensRegistration.ts

```diff
@@ -381,15 +381,18 @@ export const saveCommitRegistrationParameters = (
     ensRegistration: { registrations, currentRegistrationName },
     settings: { accountAddress },
   } = getState();
+  const registrationName =
+    (registrationParameters as RegistrationParameters)?.name ||
+    currentRegistrationName;
```

Comment:
this is used for the speed ups, you could be in the registration flow or from the tx list